openc3/lib/openc3/packets/parsers/xtce_parser.rb

```diff
@@ -178,14 +178,24 @@ def xtce_process_element(element)
       when 'ParameterTypeSet', 'EnumerationList', 'ParameterSet', 'ContainerSet',
         'EntryList', 'DefaultCalibrator', 'DefaultAlarm', 'RestrictionCriteria',
         'ComparisonList', 'MetaCommandSet', 'ArgumentTypeSet', 'ArgumentList',
-        'ArgumentAssignmentList', 'LocationInContainerInBits'
+        'ArgumentAssignmentList', 'LocationInContainerInBits', 'ReferenceTime',
+        'ErrorDetectCorrect'
         # Do Nothing
 
+      when 'CRC'
+        # TODO: Anything else here?
```

Comment:
Not sure what to do for now, probably just leave a TODO.  We could create a new CrcProcessor to check the packet in telemetry and maybe set the Crc for commands.